operator_ui/src/components/JobRuns/StatusCard.tsx

```diff
@@ -55,12 +96,24 @@ const StatusCard = ({ title, classes, children }: IProps) => {
       )}
     >
       <div className={classes.head}>
-        <StatusIcon className={classes.statusIcon} width={80}>
+        <StatusIcon
+          isForStatusCard={true}
+          className={classes.statusIcon}
+          width={80}
+        >
           {title}
         </StatusIcon>
-        <Typography className={classes.statusText} variant="h5" color="inherit">
-          {titleCase(title)}
-        </Typography>
+        <Grid container alignItems="center" className={classes.statusRoot}>
+          <Grid item xs={9}>
+            <Typography className={classes.statusText} variant="h5">
+              {titleCase(title)}
+            </Typography>
+            {renderElapsedTime(jobRun, classes)}
+          </Grid>
+          <Grid item xs={3}>
+            {title === 'completed' && renderLinkEarned(classes, jobRun)}
```

Comment:
How do you feel about rendering a component rather than a function so that we can ignore parameter order? e.g. `<EarnedLink  ... />`